<ui version="4.0" >
 <class>BulletsUi</class>
 <widget class="QWidget" name="BulletsUi" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>321</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="2" colspan="2" >
    <widget class="QComboBox" name="bulletpaneCO" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>&amp;Font:</string>
     </property>
     <property name="buddy" >
      <cstring>bulletpaneCO</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2" >
    <widget class="QLabel" name="sizeL" >
     <property name="text" >
      <string>Si&amp;ze:</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy" >
      <cstring>bulletsizeCO</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="3" >
    <widget class="QComboBox" name="bulletsizeCO" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text" >
       <string>Default</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Tiny</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Smallest</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Smaller</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Small</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Normal</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Large</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Larger</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Largest</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Huge</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Huger</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="3" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3" >
    <widget class="QLineEdit" name="customLE" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2" >
    <widget class="QCheckBox" name="customCB" >
     <property name="text" >
      <string>&amp;Custom bullet:</string>
     </property>
    </widget>
   </item>
   <item rowspan="5" row="1" column="0" >
    <widget class="QListWidget" name="levelLW" >
     <property name="maximumSize" >
      <size>
       <width>96</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item rowspan="5" row="1" column="4" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>16</width>
       <height>271</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>&amp;Level:</string>
     </property>
     <property name="buddy" >
      <cstring>levelLW</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3" >
    <widget class="QStackedWidget" name="bulletpaneSW" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>0</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>levelLW</tabstop>
  <tabstop>bulletpaneCO</tabstop>
  <tabstop>bulletpaneSW</tabstop>
  <tabstop>customCB</tabstop>
  <tabstop>customLE</tabstop>
  <tabstop>bulletsizeCO</tabstop>
 </tabstops>
 <includes>
  <include location="local" >qt_i18n.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>customCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>customLE</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>203</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel" >
     <x>319</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>203</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel" >
     <x>273</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>bulletpaneCO</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>203</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel" >
     <x>320</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customCB</sender>
   <signal>toggled(bool)</signal>
   <receiver>bulletpaneSW</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>203</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel" >
     <x>320</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
